scripts/specializations/CpCourseGeneratorSettings.lua

```diff
@@ -174,11 +164,6 @@ function CpCourseGeneratorSettings:raiseCallback(callbackStr, setting, ...)
     SpecializationUtil.raiseEvent(self, callbackStr, setting, ...)
 end
 
-function CpCourseGeneratorSettings:setAutomaticWorkWidth()
```

Comment:
This function was also use for the hud setting: ``
<Setting classType="AIParameterSettingList" name="workWidth" min="1" max="50" incremental="0.1" unit="2" setDefault="setAutomaticWorkWidth" onChangeCallback="cpShowWorkWidth"/>
``

Reply:
Might be a good idea to move these code parts into this function:
 `
local spec = self.spec_cpCourseGeneratorSettings
local width, offset = WorkWidthUtil.getAutomaticWorkWidthAndOffset(self)
spec.workWidth:setFloatValue(width)
`

Reply:
this was the fix for clicking width/offset in the HUD, should now be working fine.